### ISSUE TITLE
LDA show_topic() might show wordids, not words

### ISSUE THREAD

**Issue by bestxolodec** [RaRe-Technologies/gensim#354]
I mean this line:
https://github.com/piskvorky/gensim/blob/develop/gensim/models/ldamodel.py#L757 
should be 

```
beststr = [(topic[id], id) for id in bestn]
```

Otherwise it is not possible to get similar output to tfidf model, which shows list of tuples such as this

```
 (98801, 0.008706198772450975),
```

including   id of the word and its tfidf metric. 
Maybe such a behavior should rely on some function parameter, so it dont ruin any other related code which uses default word representation of `show_topic`.
I mean something like this:

``` (python)
    def show_topic(self, topicid, topn=10, show_ids=False):                         
        """                                                                         
        Return a list of `(words_probability, word)` 2-tuples for the most probable
        words in topic `topicid`.                                                   

        Only return 2-tuples for the topn most probable words (ignore the rest). 

        """                                                                         
        topic = self.state.get_lambda()[topicid]                                    
        topic = topic / topic.sum() # normalize to probability dist                 
        bestn = numpy.argsort(topic)[::-1][:topn]                                   
        if show_ids:                                                                
            beststr = [(topic[id], id) for id in bestn]                             
        else:                                                                       
            beststr = [(topic[id], self.id2word[id]) for id in bestn]               
        return beststr       
```

Nevertheless, even this approach is contradictory in the order of `id, metric` inside a tuple - tfidf shows id as a first parameter  and `lda.show_topics()` does it as a second.

**Comment by cscorley** [comment]
I think that is something desirable, especially making sure that the `(id, metric)` is consistent across modules. Mind opening a PR?

BTW, there's no need to change the linked line. It isn't returned in the end for that method, anyway. I think you meant a few lines up, in `show_topic`: https://github.com/piskvorky/gensim/blob/develop/gensim/models/ldamodel.py#L734

**Comment by menshikh-iv** [comment]
Resolved in #448.
